db/revert/ccbc_job_executor_permissions.sql

```diff
@@ -0,0 +1,7 @@
+-- Revert ccbc:ccbc_job_executor_permissions from pg
+
+begin;
+
+-- XXX Add DDLs here.
```

Comment:
I'm guessing we want a revert in here if we want to remove this role at one point? Looking at the old revert for `create_roles` it implies we only want to do it manually so I'm not sure, maybe @matthieu-foucault can shed some light on this

Reply:
@AntBush okay I took care of everything except for this, take a look when you get time. 

@matthieu-foucault did you have any thoughts on this comment?

Reply:
> thoughts

The comment in the old revert from `create_roles` don't necessarily apply to this project.

Reply:
I'm not sure I get the point of this, maybe if we dropped the other roles on revert but given how `create_roles` revert currently is I'm not going to bother.